source/agora/common/Transaction.d

```diff
@@ -22,6 +22,7 @@ import agora.common.Data;
 import agora.common.Hash;
 import agora.common.crypto.Key;
 
+import std.stdio;
```

Comment:
?

Reply:
It was added for test purposes and deleted.